nexus/tests/integration_tests/schema.rs

```diff
@@ -55,6 +56,9 @@ async fn test_setup<'a>(
     builder.config.pkg.schema = Some(SchemaConfig { schema_dir });
     builder.start_internal_dns().await;
     builder.start_external_dns().await;
+    let sp_conf: Utf8PathBuf = DEFAULT_SP_SIM_CONFIG.into();
+    builder.start_gateway(SwitchLocation::Switch0, None, sp_conf.clone()).await;
+    builder.start_gateway(SwitchLocation::Switch1, None, sp_conf).await;
```

Comment:
I saw this here and in another test.  Is this because starting Nexus now depends on these things, or is this just because things don't always work in Nexus when these aren't running?

Do we have a test that Nexus still starts without these?

Reply:
It's because dendrite needs MGS in order to work in the test suite (dendrite wasn't actually fully starting up in our test suite until this PR)